test/powershell/Language/Classes/scripting.Classes.inheritance.tests.ps1

```diff
@@ -449,11 +449,11 @@ Describe 'Classes inheritance ctors' -Tags "CI" {
             # we should not allow direct .ctor call.
             $o.{.ctor}()
         } catch {
-            $_.FullyQualifiedErrorId | Should Be MethodNotFound
+            $_.FullyQualifiedErrorId | Should -Be MethodNotFound
```

Comment:
this can be (lines 448-456):
{ $o.{.ctor}() } | Should -Throw -ErrorId "MethodNotFound"